CHANGELOG.md

```diff
@@ -1,22 +1,21 @@
-<!--
-SPDX-FileCopyrightText: 2020 Helmholtz Centre for Environmental Research (UFZ)
-SPDX-FileCopyrightText: 2020 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
+# Changelog
 
-SPDX-License-Identifier: Apache-2.0
--->
+## [Unreleased](https://github.com/hifis-net/ssh-keys/tree/HEAD)
 
-# Changelog
+[Full Changelog](https://github.com/hifis-net/ssh-keys/compare/v1.2.0...HEAD)
 
-All notable changes to this project will be documented in this file.
+**Merged pull requests:**
 
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+- Implement GitHub actions workflows [\#2](https://github.com/hifis-net/ssh-keys/pull/2) ([Normo](https://github.com/Normo))
 
-Group your changes into these categories:
+<!--
+SPDX-FileCopyrightText: 2020 Helmholtz Centre for Environmental Research (UFZ)
+SPDX-FileCopyrightText: 2020 Helmholtz-Zentrum Dresden-Rossendorf (HZDR)
 
-`Added`, `Changed`, `Deprecated`, `Removed`, `Fixed`, `Security`.
+SPDX-License-Identifier: Apache-2.0
+-->
```

Comment:
Dear Norman @Normo, again, this seems to be the right location for the generator to place the comment. I would move it up.

Reply:
See here https://github.com/hifis-net/ansible-role-rsd/pull/7#discussion_r847092501